module/modmanager/manager.go

```diff
@@ -1119,15 +1121,55 @@ func (mgr *Manager) FirstRun(ctx context.Context, conf config.Module) error {
 	for key, val := range moduleEnvironment {
 		cmd.Env = append(cmd.Env, key+"="+val)
 	}
-	cmdOut, err := cmd.CombinedOutput()
 
-	resultLogger := logger.With("path", firstRunPath, "output", string(cmdOut))
+	stdOut, err := cmd.StdoutPipe()
 	if err != nil {
-		resultLogger.Errorw("command failed", "error", err)
 		return err
 	}
+	stdErr, err := cmd.StderrPipe()
+	if err != nil {
+		return err
+	}
+
+	var combined []string
+	var mu sync.Mutex
 
-	resultLogger.Infow("command succeeded")
+	scanOut := bufio.NewScanner(stdOut)
+	go func() {
+		for scanOut.Scan() {
+			out := scanOut.Text()
+			logger.Infow("first run stdio", "output", out)
+			mu.Lock()
+			combined = append(combined, out)
+			mu.Unlock()
+		}
+	}()
+	scanErr := bufio.NewScanner(stdErr)
+	go func() {
+		for scanErr.Scan() {
+			out := scanErr.Text()
+			logger.Warnw("first run stderr", "output", out)
```

Comment:
why warn and not just err?

Reply:
Warn felt appropriate to me. All kind of "not errors" go to stderr. And so long as the script returns an exit code of 0, we're going to mark it as a success. It might be surprising if there was a bunch of "ERROR" logs and that end up as a successful run.

Reply:
yep, @dgottlieb's comment pretty much summarizes my thoughts on this. I can add a code comment with my rationale if that helps